tests/tokens.test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from 'chai'
+import { Mappings } from './wasm'
+import { ValueKind } from './wasm/runtime/ethereum'
+
+describe('onTokenListing', function () {
+  it('creates a token', async () => {
+    const mappings = await Mappings.load()
+    mappings.setCallHandler((call) => {
```

Comment:
Cool, I assume in then long term we can just pass in the Abis and perform the calls automatically?

Reply:
Correct! The idea is to have something like we have for the events and entities, except for events. Ultimately, I think this should be auto-generated based on the `subgraph.yaml` definition.